site/gatsby-site/migrations/2023.05.15T11.49.00.csetv1-remove-ai-qualifier.js

```diff
@@ -0,0 +1,2033 @@
+const config = require('../config');
+
+/** @type {import('umzug').MigrationFn<any>} */
+exports.up = async ({ context: { client } }) => {
+  const taxaCollection = client.db(config.realm.production_db.db_name).collection('taxa');
+
+  await taxaCollection.deleteOne({ namespace: 'CSETv1' });
+
+  await taxaCollection.insertOne(csetV1TaxaEntry);
+
+  for (let i = 1; i < 4; i++) {
+    const namespace = 'CSETv1_Annotator-' + i;
+
+    await taxaCollection.deleteOne({ namespace });
+
+    await taxaCollection.insertOne({
+      ...csetV1TaxaEntry,
+      namespace,
+      _id: undefined,
+    });
+  }
+
+  const classificationsCollection = client
+    .db(config.realm.production_db.db_name)
+    .collection('classifications');
+
+  const classifications = await classificationsCollection.find({}).toArray();
+
+  for (const classification of classifications) {
+    classificationsCollection.updateOne(
+      { namespace: classification.namespace, incident_id: classification.incident_id },
+      {
+        $set: {
+          attributes: classification.attributes.map((attribute) => {
+            let short_name = attribute.short_name;
+
+            if (short_name == 'Clear link to AI') {
```

Comment:
@lmcnulty looks like this change is breaking this test https://app.netlify.com/sites/staging-aiid/deploys/648923203cd16b0008c3dfbd#L532 (which I rewritten for clarity) .

Does the taxonomy definition need to be updated too?

Reply:
I just noticed that one is `technology` and another is `Technology` 🤔